demo/clusters/kind/scripts/build-kind-image.sh

```diff
@@ -29,6 +29,10 @@ if [ "${EXISTING_IMAGE_ID}" != "" ]; then
 	exit 0
 fi
 
+if [[ x"${KIND_IMAGE_BASE}" == x"" && x"${KIND_K8S_TAG}" == x"" ]]; then
+    exit 0
+fi
+
```

Comment:
Ia tehre a reason to use the double `[[` syntax and the `x` before the strings here? Mostly because it is inconsistent with the rest of the bash scripts here.

Reply:
No, not really. I'm just used to the double syntax. Will update.